### PR TITLE
Allow to override volume disk size

### PR DESCRIPTION
**Pull request by mbornoz**

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -39,6 +39,12 @@ variable "server_memory" {
   default     = 1024
 }
 
+variable "server_volume_size" {
+  description = "The size of the volume in bytes attached to the server"
+  type        = string
+  default     = "10737418240"
```
Comment (mcanevet) [inline variables.tf]: Why not a `number`?

File: variables.tf [hunk 2]
```diff
@@ -50,3 +56,9 @@ variable "agent_memory" {
   type        = number
   default     = 1024
 }
+
+variable "agent_volume_size" {
+  description = "The size of the volume in bytes attached to the agents"
+  type        = string
+  default     = "10737418240"
```
Comment (mcanevet) [inline variables.tf]: Same here

**Review by mcanevet — CHANGES_REQUESTED** [review]

File: variables.tf (hunk 1 above)
Comment (mbornoz) [reply variables.tf]: I asked myself the same question but in the end I stupidly copied the existing format

Comment (mbornoz) [reply variables.tf]: @mcanevet I just tested/updated this PR with numbers, it seems to work as expected

**Review by mcanevet — APPROVED** [review]

**Pull request merged**